### PR TITLE
MOL 298 - refunds generated from Mollie orders are not populating in Sylius + MOL 294 & MOL 299

### PR DESCRIPTION
**Pull request by marekrzytki**

### CODE REVIEW

File: spec/Checker/Refund/MollieOrderRefundCheckerSpec.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+declare(strict_types=1);
+
+namespace spec\BitBag\SyliusMolliePlugin\Checker\Refund;
+
+use BitBag\SyliusMolliePlugin\Checker\Refund\MollieOrderRefundChecker;
+use BitBag\SyliusMolliePlugin\Checker\Refund\MollieOrderRefundCheckerInterface;
+use Mollie\Api\Resources\Order;
+use PhpSpec\ObjectBehavior;
+
+final class MollieOrderRefundCheckerSpec extends ObjectBehavior
```
Comment (senghe) [inline spec/Checker/Refund/MollieOrderRefundCheckerSpec.php:19]: I would add the scenario with lines [0, 1] and [1, 0] also.

File: src/Validator/Refund/RefundUnitsCommandValidator.php [hunk 2]
```diff
@@ -47,6 +47,10 @@ public function validate(RefundUnits $command): void
             throw OrderNotAvailableForRefunding::withOrderNumber($command->orderNumber());
         }
 
+        if (empty($command->units()) && empty($command->shipments())) {
+            throw new OrderNotAvailableForRefunding(sprintf('There is no units to refund in order %s', $command->orderNumber()));
```
Comment (senghe) [inline src/Validator/Refund/RefundUnitsCommandValidator.php:51]: Is this a public exception? If so, we should translate the message

**Review by senghe — CHANGES_REQUESTED** [review]

File: spec/Checker/Refund/MollieOrderRefundCheckerSpec.php (hunk 1 above)
Comment (marekrzytki) [reply spec/Checker/Refund/MollieOrderRefundCheckerSpec.php:19]: Done

File: src/Validator/Refund/RefundUnitsCommandValidator.php (hunk 2 above)
Comment (marekrzytki) [reply src/Validator/Refund/RefundUnitsCommandValidator.php]: It's not public, but I found typo

**Pull request merged**